src/herbivoresAndCarnivores.js

```diff
@@ -28,7 +28,7 @@ class Carnivore extends Animal {
 
   bite(victim) {
     if (victim.__proto__ === Herbivore.prototype
-    && victim.hidden === false) {
+    && !victim.hidden) {
       victim.health -= 50;
     }
     Animal.alive = Animal.alive.filter((animal) => animal.health > 0);
```

Comment:
if the victim does not die, this code performs an unnecessary action, I suggest adding an `if` construct that checks if the animal has died =)